warp10/src/main/java/io/warp10/fdb/FDBSIZE.java

```diff
@@ -34,8 +36,18 @@ public FDBSIZE(String name) {
   @Override
   public Object apply(WarpScriptStack stack) throws WarpScriptException {
 
-    if (null == Capabilities.get(stack, FDBUtils.CAPABILITY_ADMIN)) {
-      throw new WarpScriptException(getName() + " missing capability.");
+    String regexp;
+
+    if (null != Capabilities.get(stack, FDBUtils.CAPABILITY_ADMIN)) {
+      regexp = null;
+    } else if (null == Capabilities.get(stack, FDBUtils.CAPABILITY_SIZE)) {
+      regexp = Capabilities.get(stack, FDBUtils.CAPABILITY_SIZE);
+      // Empty string means match all tenants
+      if ("".equals(regexp)) {
+        regexp = null;
+      }
+    } else {
+      throw new WarpScriptException(getName() + " missing '" + FDBUtils.CAPABILITY_SIZE + "' or '" + FDBUtils.CAPABILITY_ADMIN + "' capability.");
```

Comment:
if you do not have CAPABILITY_ADMIN nor CAPABILITY_SIZE, you can read everything !